lepton/openstack_instance.go

```diff
@@ -37,10 +37,16 @@ func getOpenStackInstances(provider *gophercloud.ProviderClient, opts servers.Li
 
 		for _, s := range serverList {
 			if val, ok := s.Metadata["CreatedBy"]; ok && val == "ops" {
-				// fugly
 				ipv4 := ""
-				// For some instances IP is not assigned.
+
+				// addresses may have a different structure in each cloud provider
+				// vexx
```

Comment:
not ideal but i suppose this is fine for now

Reply:
Yes, openstack should uniformize this. My guess is that one way is openstack API v2 an the other is API v3.